src/status_im2/contexts/quo_preview/preview.cljs

```diff
@@ -108,6 +108,7 @@
          :show-cancel    false
          :style          {:border-radius 4
                           :border-width  1
+                          :color         (colors/theme-colors colors/neutral-100 colors/white)
```

Comment:
Please elaborate

Reply:
When the dark theme is set the color of the text is black and you cannot differentiate it from the background. Now it works properly.